pkgs/development/python-modules/static3/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, fetchPypi, buildPythonPackage, pytestcov, webtest }:
+
+buildPythonPackage rec {
+  pname = "static3";
+  version = "0.7.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1j275ys1byh8w55nginyjflxr4rmgrzfy2xj5sphfmf79g342ik7";
+  };
```

```suggestion
  src = fetchFromGitHub {
    owner = "rmohr";
    repo = pname;
    rev = "refs/tags/v${version}";
    hash = "sha256-uFgv+57/UZs4KoOdkFxbvTEDQrJbb0iYJ5JoWWN4yFY=";
  };
```

it's common that many pypi distributions do not include test related file. So fetch it from github will fix test not found error.

---

Fixed! Thanks!